Mathlib/CategoryTheory/GradedObject/Unitor.lean

```diff
@@ -252,6 +252,105 @@ lemma mapBifunctorRightUnitor_naturality :
 
 end RightUnitor
 
+section
+
+variable {I₁ I₂ I₃ J : Type*} [Zero I₂]
+
+/-- Given two maps `r : I₁ × I₂ × I₃ → J` and `π : I₁ × I₃ → J`, this structure is the
+input in the formulation of the triangle equality `mapBifunctor_triangle` which
+relates the left and right unitor and the associator for `GradedObject.mapBifunctor`. -/
+structure TriangleIndexData (r : I₁ × I₂ × I₃ → J) (π : I₁ × I₃ → J) where
+  /-- a map `I₁ × I₂ → I₁` -/
+  p₁₂ : I₁ × I₂ → I₁
+  hp₁₂ (i : I₁ × I₂ × I₃) : π ⟨p₁₂ ⟨i.1, i.2.1⟩, i.2.2⟩ = r i
```

Comment:
It doesn't matter, but you're using both `⟨⟩` and `()` for products in this file. You could clean this up to aid readability slightly.